bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.html

```diff
@@ -81,8 +81,18 @@
               <div fxFlex fxLayout="row" fxLayoutAlign="flex-start center" fxLayoutGap="15px">
                 <div class="mt-1 mb-1 sequence-title" fxLayout="row" fxLayoutAlign="start center">
                   <p class="m-0">
-                    <span class="bold" [textContent]="currentSequence.name"></span>&nbsp;
-                    <span [textContent]="currentSequence.getStatus()"></span>
+                    <span class="bold" [textContent]="currentSequence.name"></span>
+                    of
+                    <span
+                      class="bold"
+                      uitestid="keptn-sequence-view-serviceName"
+                      [textContent]="
+                        currentSequence.getShortImageName()
```

Comment:
Just a minor code improvement.
```ts
currentSequence.getShortImageName() || currentSequence.service
```
instead of
```ts
currentSequence.getShortImageName() ? currentSequence.getShortImageName() : currentSequence.service
```
to prevent executing `getShortImageName` twice.

Reply:
Thanks! I've made the suggested improvement: https://github.com/keptn/keptn/pull/7283/commits/d7ca1f059f33ee9653d31da4988f5387d3318e89